src/odemis/driver/pwrmccdaq.py

```
@@ -63,7 +63,7 @@ class MCCDevice(HwComponent):
     This class can be inherited for more functionality to support more complex components.
     """
     def __init__(self, name: str, role: str, mcc_device: Optional[str],
-                 di_channels: Dict[int, Tuple[str, bool]] = {}, **kwargs):
+                 di_channels: Dict[int, Tuple[str, bool]] = {}, children: Dict[str, dict] = {}, daemon=None, **kwargs):
```

Ruff (0.15.18)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)